<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-LSIEJKN" ignored="0" name="Report" tests="5" failures="2" timestamp="2020-11-11T10:07:55 CST" time="0.039" errors="0">
  <testcase name="test1" time="0.011" classname="com.webtest.demo.Report"/>
  <testcase name="test2" time="0.004" classname="com.webtest.demo.Report"/>
  <testcase name="test3" time="0.001" classname="com.webtest.demo.Report">
    <failure type="java.lang.AssertionError" message="expected [1] but found [13]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [13]
at com.webtest.demo.Report.test3(Report.java:22)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test4" time="0.0" classname="com.webtest.demo.Report"/>
  <testcase name="test5" time="0.004" classname="com.webtest.demo.Report">
    <failure type="java.lang.AssertionError" message="expected [11] but found [13]">
      <![CDATA[java.lang.AssertionError: expected [11] but found [13]
at com.webtest.demo.Report.test5(Report.java:34)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test5 -->
</testsuite> <!-- Report -->
